ligand/templates/target_details.html

```diff
@@ -469,16 +596,25 @@
             var checked_data = table_affinity.rows('.alt_selected').data();
             var checked_data = table_affinity.rows('.alt_selected').data();
             var csv_data = [];
-            csv_data.push('ChEMBL ID;Receptor;Species;Purchasable;No. records;Assay type;Min.;Average;Max.;Unit;Mol. weight;Rot. Bonds;H don;H acc;LogP;Smiles')
+           
+            {% if mode == 'extended' %}
+              csv_data.push('Lig. Name;GPCRdb ID;#Vendors;Ref. Ligand;Fold selectivity (Affinity);# Tested GPCRs;Species;p-value (-log);Value type;Activ. relation;Activ. value;Assay type;Assay descript.;Source; Mol. weight;Rot. Bonds;H don;H acc;LogP;Smiles;DOI')
+            {% endif %}
+            {% if mode == 'compact' %}
+              csv_data.push('Lig. Name;GPCRdb ID;Lig. type;Ref. Ligand;Fold selectivity (Affinity);# Tested GPCRs;{% if ligand_query %}Receptor (Gtp);Receptor (UniProt);{% endif %}Species;{% if ligand_query %}Class;Family;{% endif %}#Vendors;Assay type;#Records;Min.;Average;Max.;Activ. Value;Source;Mol. weight;Rot. Bonds;H don;H acc;LogP;Smiles')
+            {% endif %}
+            
             for (i = 0; i < checked_data.length; i++) {
                 var csv_row = []
                 for (j = 1; j < checked_data[i].length; j++) {
                     var div = document.createElement("div");
                     div.innerHTML = checked_data[i][j];
                     if (typeof div.innerText !== "undefined") {
-                        csv_row.push(div.innerText.replace(/\s+/g, ''));
+                      //csv_row.push(div.innerText.replace(/\s+/g, ''));
+                      csv_row.push(div.innerText.trim());
                     } else {
-                        csv_row.push(div.textContent.replace(/\s+/g, ''));
+                        //csv_row.push(div.textContent.replace(/\s+/g, ''));
```

Comment:
please remove commented line

Reply:
Solved in [bb9beb8](https://github.com/protwis/protwis/pull/550/commits/bb9beb8c3231460510e199f66a17a1c512c6c8e0).